app/src/main/java/com/ddoczi/tasky/authentication/presentation/composables/BaseTextField.kt

```diff
@@ -0,0 +1,61 @@
+package com.ddoczi.tasky.authentication.presentation.composables
+
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.tooling.preview.Preview
+import com.ddoczi.tasky.authentication.enums.InputFieldType
+import com.ddoczi.tasky.ui.theme.TaskyTheme
+
+@Composable
+fun BaseTextField(
```

Comment:
Same here, I'd prefer something like `TaskyTextField` or `StandardTextField`. Base also kind of makes me feel like there's a hierarchy and this is the root text field of all others